keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -190,8 +192,10 @@ spec:
   generateBucketName: [4]
   bucketClassName: [5]
   secretName: [6]
+  requestProtocol: [7]
```

Comment:
why are we using "request" in these 2 names? Aren't `protocol` and `accessModes` (plural?) ok?

Reply:
I'm not in love with it either TBH.  It _seemed_ more descriptive of what they represent, but then it may not be in the philosophy of the nomenclature.

Reply:
Right. The philosophy part is the reason we're suggesting `Bucket` rather than `BucketRequest`.